package.json

```diff
@@ -2,8 +2,21 @@
   "name": "healthz-prober",
   "version": "1.0.0",
   "description": "probes healthz endpoints and reports to slack",
-  "main": "index.ts",
+  "main": "src/index.ts",
   "scripts": {
+    "start": "npm run build && node --env-file=.env dist/index.js",
+    "start:watch": "node --env-file=.env --watch dist/index.js",
+    "build": "tsc",
+    "build:watch": "tsc -w",
+    "dev": "(start npm run build:watch || npm run build:watch) & npm run start:watch",
```

Comment:
Running dev mode needed a different approach on windows: doesn't show the output.